document/src/vespa/document/config/documentmanager.def

```diff
@@ -5,9 +5,6 @@ namespace=document.config
 ## Whether attempts to set an undefined field should be ignored rather than causing an error
 ignoreundefinedfields bool default=false
 
-## Whether to enable compression in this process.
-enablecompression bool default=false
```

Comment:
It is safe to remove a parameter used by older model currently in use (hosted/cloud)?

Reply:
Yes, for our own config that shoul be fine. They belong to the model.